### PR TITLE
Maven 4.0.0-alpha-13 release notes

### PR DESCRIPTION
**Pull request by cstamas**
Release notes for 4.0.0-alpha-13.

From where to get XSDs? Were they generated on the release machine (as now am not at it)?
Also, the contents are TBD yet.

### CODE REVIEW

**Comment by gnodet** [comment]
> Release notes for 4.0.0-alpha-13.
> 
> From where to get XSDs? Were they generated on the release machine (as now am not at it)? Also, the contents are TBD yet.

On the release computer, you should have the following:
```
./maven-embedder/target/generated-site/resources/xsd/core-extensions-1.2.0.xsd
./maven-model/target/generated-site/resources/xsd/maven-4.1.0.xsd
./api/maven-api-plugin/target/generated-site/resources/xsd/lifecycle-1.0.0.xsd
./api/maven-api-plugin/target/generated-site/resources/xsd/plugin-2.0.0.xsd
./api/maven-api-settings/target/generated-site/resources/xsd/settings-1.3.0.xsd
```
What I did for alpha-12 is that I published those with an `alpha-12` suffix in the website and modified the `.htaccess` to redirect to those ones.  This is obviously a step that will go away when moving to GA because the XSDs would then be actually released and new versions created if needed.
See https://github.com/apache/maven-site/commit/ebfcd5042f46f40a99a20e98c1792245dbcec154

**Comment by cstamas** [comment]
I have following:
```
[cstamas@blondie checkout ((maven-4.0.0-alpha-13))]$ find . -type f | grep generated-site | grep .xsd
./api/maven-api-plugin/target/generated-site/resources/xsd/lifecycle-1.0.0.xsd
./api/maven-api-plugin/target/generated-site/resources/xsd/plugin-2.0.0.xsd
./api/maven-api-settings/target/generated-site/resources/xsd/settings-1.3.0.xsd
./maven-compat/target/generated-site/resources/xsd/profiles-1.0.0.xsd
./maven-compat/target/generated-site/resources/xsd/paramdoc-1.0.0.xsd
./maven-embedder/target/generated-site/resources/xsd/core-extensions-1.2.0.xsd
./maven-model/target/generated-site/resources/xsd/maven-4.1.0.xsd
./maven-repository-metadata/target/generated-site/resources/xsd/repository-metadata-1.2.0.xsd
./maven-settings/target/generated-site/resources/xsd/settings-2.0.0.xsd
./maven-toolchain-model/target/generated-site/resources/xsd/toolchains-1.1.0.xsd
[cstamas@blondie checkout ((maven-4.0.0-alpha-13))]$ 
```

Things like `maven-repository-metadata-1.2.0.xsd` is not even on site... do I need to move all these (and prefix alpha-13?)

**Comment by cstamas** [comment]
And now I don't get something: at least looks fishy to me: when I compare repository-metadata 1.1.0 (current metadata) and 1.2.0 (generated by Maven4) I see some changes. Most of them are "comment text changes", BUT there are some ordering changes as well (but no structural). 

So Maven4 deploys repository-metadata 1.2.0? What will Maven3 do with it? But again, I see no structural changes between XSD 1.1.0 and 1.2.0... structure wise they look same to me, unless "version=1.2.0" does not makes Maven3 belly up. Should not this simply remain model version 1.1.0?

**Comment by cstamas** [comment]
Checked Maven 3 (3.9.x branch), to me it seems it does not care for version of repository-metadata...

**Comment by gnodet** [comment]
> I have following:
> 
> ```
> [cstamas@blondie checkout ((maven-4.0.0-alpha-13))]$ find . -type f | grep generated-site | grep .xsd
> ./api/maven-api-plugin/target/generated-site/resources/xsd/lifecycle-1.0.0.xsd
> ./api/maven-api-plugin/target/generated-site/resources/xsd/plugin-2.0.0.xsd
> ./api/maven-api-settings/target/generated-site/resources/xsd/settings-1.3.0.xsd
> ./maven-compat/target/generated-site/resources/xsd/profiles-1.0.0.xsd
> ./maven-compat/target/generated-site/resources/xsd/paramdoc-1.0.0.xsd
> ./maven-embedder/target/generated-site/resources/xsd/core-extensions-1.2.0.xsd
> ./maven-model/target/generated-site/resources/xsd/maven-4.1.0.xsd
> ./maven-repository-metadata/target/generated-site/resources/xsd/repository-metadata-1.2.0.xsd
> ./maven-settings/target/generated-site/resources/xsd/settings-2.0.0.xsd
> ./maven-toolchain-model/target/generated-site/resources/xsd/toolchains-1.1.0.xsd
> [cstamas@blondie checkout ((maven-4.0.0-alpha-13))]$ 
> ```
> 
> Things like `maven-repository-metadata-1.2.0.xsd` is not even on site... do I need to move all these (and sufix alpha-13?)

Is it a public resource ? I guess we don’t know for sure, so maybe publish it as alpha-13. AFAIK, old readers and writers don’t care about the namespace, but the new ones do (at least, it is reported in the root element).

**Comment by gnodet** [comment]
> And now I don't get something: at least looks fishy to me: when I compare repository-metadata 1.1.0 (current metadata) and 1.2.0 (generated by Maven4) I see some changes. Most of them are "comment text changes", BUT there are some ordering changes as well (but no structural).
> 
> So Maven4 deploys repository-metadata 1.2.0? What will Maven3 do with it? But again, I see no structural changes between XSD 1.1.0 and 1.2.0... structure wise they look same to me, unless "version=1.2.0" does not makes Maven3 belly up. Should not this simply remain model version 1.1.0 w/ "updated comments/descriptions"?

The 1.2.0 comes from the new immutable model afaik, there’s a need to switch to a different version so that we can have both v3 and v4 models. I don’t think there’s any real change to the schema though.

**Comment by cstamas** [comment]
@gnodet pls review, pused XSDs and redirects.

**Comment by cstamas** [comment]
Seems legit, as we can tell the difference :smile: "Generated by Modello 2.3.0".
But, now spotted other difference: older XSD have ASL2 headers, while new one does not have those. Were they present in templates or you just added them by hand?

**Comment by gnodet** [comment]
> Seems legit, as we can tell the difference 😄 "Generated by Modello 2.3.0". But, now spotted other difference: older XSD have ASL2 headers, while new one does not have those. Were they present in templates or you just added them by hand?

Yes.

**Comment by cstamas** [comment]
Ping? Announce is waiting to have this merged, pls eyeball

File: content/markdown/docs/4.0.0-alpha-13/release-notes.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+- the web site: [https://maven.apache.org/][2]
+- the maven-user mailing list: [https://maven.apache.org/mailing-lists.html](/mailing-lists.html)
+- the reference documentation: [https://maven.apache.org/ref/4.0.0-alpha-13/](/ref/4.0.0-alpha-13/)
+
+## Overview About the Changes
+
+The full list of changes can be found in our [issue management system][4].
+
+Notable changes include:
+* requires Java 17 as "minimum runtime Java requirement"
```
Comment (slawekjaranowski) [inline content/markdown/docs/4.0.0-alpha-13/release-notes.md:46]: I only would like to bold `Java 17` it is important change, or add at the beginning that it is first release with Java 17.

File: content/markdown/docs/history.md.vm [hunk 2]
```diff
@@ -81,7 +81,8 @@ Date format is: YYYY-MM-DD
 <!--  release( date version announce latest-minor jdk rowspan ) -->
 <!-- The following two lines needs to be fixed in x days if the archive picks up the announcement mail. -->
 
-#release( "2024-01-12" "4.0.0-alpha-12" "https://lists.apache.org/thread/8nq1mssngrol0zzotpyyll2rdm8f77ob" "true" "Java 8" "9" )
+#release( "2024-03-06" "4.0.0-alpha-13" "https://lists.apache.org/thread/8nq1mssngrol0zzotpyyll2rdm8f77ob" "true" "Java 17" "1" )
```
Comment (slawekjaranowski) [inline content/markdown/docs/history.md.vm:84]: https://lists.apache.org/thread/p64svxy8tf5fblmlo7k2zkbblml0pbo2

File: content/markdown/docs/history.md.vm [hunk 3]
```diff
@@ -81,7 +81,8 @@ Date format is: YYYY-MM-DD
...
+#release( "2024-01-12" "4.0.0-alpha-12" "https://lists.apache.org/thread/8nq1mssngrol0zzotpyyll2rdm8f77ob" "" "Java 8" "9" )
```
Comment (slawekjaranowski) [inline content/markdown/docs/history.md.vm:85]: https://lists.apache.org/thread/kr5ps9xgg8yhz5hxtq762pjv25jckf3z

**Review by khmarbaise — APPROVED** [review]

**Review by slawekjaranowski — APPROVED** [review]

**Pull request merged**